recipes/tests.rb

```diff
@@ -308,6 +308,68 @@ module load intelmpi && mpirun --help | grep '#{node['cluster']['intelmpi']['kit
   end
 end
 
+###################
+# instance store
+###################
+bash 'test instance store' do
+  cwd Chef::Config[:file_cache_path]
+  code <<-EPHEMERAL
+      set -xe
+      EPHEMERAL_DIR="#{node['cluster']['ephemeral_dir']}"
+
+      function set_imds_token(){
+        if [ -z "${IMDS_TOKEN}" ];then
+          IMDS_TOKEN=$(sudo curl --retry 3 --retry-delay 0 --fail -s -f -X PUT -H "X-aws-ec2-metadata-token-ttl-seconds: 900" http://169.254.169.254/latest/api/token)
+          if [ "${?}" -gt 0 ] || [ -z "${IMDS_TOKEN}" ]; then
+            echo '[ERROR] Could not get IMDSv2 token. Instance Metadata might have been disabled or this is not an EC2 instance.'
+            exit 1
+          fi
+        fi
+      }
+      function get_meta() {
+          local IMDS_OUT=$(sudo curl --retry 3 --retry-delay 0 --fail -s -q -H "X-aws-ec2-metadata-token:${IMDS_TOKEN}" -f http://169.254.169.254/latest/${1})
+          echo -n "${IMDS_OUT}"
+      }
+      function print_block-device-mapping(){
```

Comment:
minor: is it `print_block_device_mapping`?

Reply:
no, it was `print_block-device-mapping` but I have renamed it to use underscore